api/mil.py

```diff
@@ -80,3 +80,78 @@ def delete_transaction_mil(transaction_id,
         }
     )
     return response
+
+
+def post_merchant_create_transaction_mil(initiative_id,
```

Comment:
Is this method already present at line 10 of this file?

Reply:
Yes, this is the generic one introduced in this sprint, I will remove the previous one